un-needed files in published package

Hi,

I'm currently using your component. However there are a couple of issues, the first being that the `.git` folder is published to npm, this is actually breaking any subsequent `npm i` calls with the following error:

```
code EISGIT
npm ERR! git /redacted/node_modules/vue-avatar-component: Appears to be a git repo or submodule.
npm ERR! git    /redacted/node_modules/vue-avatar-component
npm ERR! git Refusing to remove it. Update manually,
npm ERR! git or move it out of the way first.
```
So this is kind of annoying and also the fact that the `.git` folder & `images` and a substantial size to the package for no real added value

I would suggest added an `.npmignore` file so as not to publish these files:

```
.git
img
```

I'm happy to open a PR with my suggested changes just let me know

--- a/.npmignore
+++ b/.npmignore
@@ -1,0 +1,4 @@
+*
+!Avatar.vue
+!readme.md
+!package.json

